features/step_definitions/admin_auction_form_steps.rb

```diff
@@ -90,7 +90,7 @@
 
 Then(/^I should see the updated delivery deadline$/) do
   within('.estimated-delivery-date') do
-    delivery_date = DefaultDateTime.new(5.business_days.after(DcTimePresenter.convert(3.business_days.from_now))).convert
+    delivery_date = DefaultDateTime.new(5.business_days.after(@end_day)).convert
```

Comment:
ok, well I don't mind this reverting change then because at least we now know that this was neither the fix nor the reason it was broken ;)